### ISSUE TITLE
Customized user-option ignores template from config

### ISSUE THREAD

**Issue by levipeto** [PH7-Jack/livewire-ui#524]
Hello, 

I hope the title makes sense. Here are some instructions to reproduce it.

1, Check wireui.php to have:
```
'select.user-option' => [
            'class' => Components\Select\UserOption::class,
            'alias' => 'select.user-option',
        ],
```

2, make changes in wireui::components.select.user-option like comment out the follwing:
`{{--        <img src="{{ data_get($option, 'src', $src) }}" class="shrink-0 h-6 w-6 object-cover rounded-full">--}}`

3, paste this code from the docs to your blade file:
```
<x-select
        label="Select Relator"
        placeholder="Select relator"
        wire:model.defer="model"
    >
        <x-select.user-option src="https://via.placeholder.com/500" label="People 1" value="1" />
        <x-select.user-option src="https://via.placeholder.com/500" label="People 2" value="2" />
        <x-select.user-option src="https://via.placeholder.com/500" label="People 3" value="3" />
        <x-select.user-option src="https://via.placeholder.com/500" label="People 4" value="4" />
    </x-select>
```
    
 4. Check result that you can see the changes: No images in the list! Great!
 
5. paste this code from the docs to your blade file:
```
<x-select
        label="Search a User"
        wire:model.defer="asyncSearchUser"
        placeholder="Select some user"
        :async-data="route('api.users.index')"
        :template="[
            'name'   => 'user-option',
            'config' => ['src' => 'profile_image']
        ]"
        option-label="name"
        option-value="id"
        option-description="email"
    />
```
6. Check that it does not show the changes; the images are back. Seems like they are hardcoded in javascript and not using the template from the config?  

**Dependencies**
 - WireUI Version: 1.17.2
 - Laravel Version: 9.34.0
 - Alpine.js Version: [3.X.X](https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js)
 - Livewire Version: 2.10
 - Lang: en_US

**Desktop**
 - OS: macOS Monterey
 - Browser: chrome
 - Version 107.0.5304.87 (Official Build) (x86_64)

**Comment by levipeto** [comment]
I searched for the <img ...> tag for the profile_image and commented them out, but there is one left in wireui.js. Seems like it's pulling the user-option template from there instead from the select.user-option in wireui.php

![image](https://user-images.githubusercontent.com/4574216/201164430-4f72b02d-0abc-40e1-8eff-a3f6eb519515.png)

**Comment by ph7jack** [comment]
@levipeto I understand, but it's not a bug. The blade component for populating the select with text and image is an option to use/customize manually. The automatic template can't be modified because it has some enhancements to work better than the blade component.
If you don't want to use an image, just use the select option. 
Thanks for using the wireui

**Comment by levipeto** [comment]
Hi Jack, thank you for your reply. I just had time to revisit this issue and tried to come up with a solution.
I tried to use the select option with @foreach but then I don't have the search.
I would prefer :async-data anyway, but it then forces the built in template that I can't overwrite.
I created my own components and registered them in config/wireui.php but it ignores them.
Why can't I just create a template or a component that I can format however I want to?
I tried to find examples but I feel lost.
